gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestPlanMapper.java

```diff
@@ -173,26 +172,33 @@ private Optional<Extension> extractAuthorisationExpiryDate(RCMRMT030101UKAuthori
     }
 
     private String extractTermText(RCMRMT030101UKDiscontinue discontinue) {
-
-        var stringBuilder = new StringBuilder();
+        String originalText = "";
         if (discontinue.hasCode() && discontinue.getCode().hasOriginalText()) {
-            var originalText = discontinue.getCode().getOriginalText();
-            if (StringUtils.isNotEmpty(originalText)) {
-                stringBuilder
-                        .append('(')
-                        .append(originalText)
-                        .append(") ");
-            }
+            originalText = discontinue.getCode().getOriginalText();
         }
 
+
         var pertinentInfo = discontinue.getPertinentInformation()
                 .stream()
                 .map(RCMRMT030101UKPertinentInformation2::getPertinentSupplyAnnotation)
                 .map(RCMRMT030101UKSupplyAnnotation::getText)
                 .filter(StringUtils::isNotBlank)
-                .collect(Collectors.joining(", "));
+                .toList();
+
+        var stringBuilder = new StringBuilder();
+
+        if (StringUtils.isNotEmpty(originalText) && !pertinentInfo.contains(originalText)) {
```

Comment:
Could the `if (discontinue.hasCode() && discontinue.getCode().hasOriginalText()) {` from line 176 be used here instead, removing the need for the `originalText` variable?